src/templates/admin/elements/core/author_dashboard.html

```diff
@@ -99,7 +99,14 @@ <h2>Published Articles</h2>
                             <td>{{ article.get_stage_display }}</td>
                             <td>{{ article.date_published }}</td>
                             <td>
-                                <a class="small button" href="{{ article.url }}">View Live</a>
+                                <a class="small button" href="{% url 'core_dashboard_article' article.pk %}">History</a> 
+                                {% for revision in article.active_revision_requests %}
+                                    {% if not revision.date_completed %}
+                                        <a class="small button" href="{% url 'do_revisions' article.pk revision.pk %}">Revision
+                                            Request</a>
+                                    {% endif %}
+                                {% endfor %}
```

Comment:
I think this is not the right thing to check for and display here. It just checks for in-progress revision requests, but in most cases there won't be any of those by the time the article is published. 

It should get completed requests, i.e. `article.completed_revision_requests`.

I think it should also check for completed copyediting reviews, via a new function that doesn't yet exist called `article.completed_author_copyedits`, based on `article.active_author_copyedits`. Together these will mirror what is happening here for active requests:

https://github.com/BirkbeckCTP/janeway/blob/bc2635fab46f45873860c78a2324f241b0bfa13f/src/templates/admin/elements/core/author_dashboard.html#L59-L69

However, the views controlling those links expect that the author only ever views an active request. They raise a 404 if the request has been completed:

https://github.com/BirkbeckCTP/janeway/blob/3f49d4b36d19a7591452461837b498e232b3e2a6/src/review/views.py#L1942-L1948

https://github.com/BirkbeckCTP/janeway/blob/3f49d4b36d19a7591452461837b498e232b3e2a6/src/copyediting/views.py#L718-L723

So we need to modify those views and think through what the author should see when viewing a completed request.

The one other complication is there could be more than one revision request or more than one copyediting review, so we need to alter the labels so the user can tell them apart. Maybe add the ID of the review after the label?

![image](https://github.com/user-attachments/assets/5cb139d5-a2b1-47dd-833e-074ee072f5a1)